auth-service/src/main/kotlin/uk/gov/communities/delta/auth/utils/EmailService.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.communities.delta.auth.utils
+
+import org.slf4j.LoggerFactory
+import org.thymeleaf.TemplateEngine
+import org.thymeleaf.context.Context
+import uk.gov.communities.delta.auth.plugins.makeTemplateResolver
+import java.util.*
+import javax.mail.*
+import javax.mail.internet.*
+
+class EmailService() {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    // TODO - what stuff needs encoding here before sending or is that done earlier?
+
+    fun sendTemplateEmail(
+        template: String,
+        toEmail: String,
+        fromEmail: String,
+        fromName: String,
+        replyToEmail: String,
+        replyToName: String,
+        subject: String,
+        mappedValues: Map<String, String>,
+    ) {
+        val templateEngine = TemplateEngine()
+        templateEngine.setTemplateResolver(makeTemplateResolver(false))
+        val context = Context(Locale.getDefault(), mappedValues)
+        val content = templateEngine.process(template, context)
+        try {
+            println("Emails not set up yet but will send an email to $toEmail with subject $subject and content $content from $fromEmail ($fromName) with replies going to $replyToEmail ($replyToName)")
+//            TODO - set this up to send emails
```

Comment:
Will involve passing in configuration and credentials similar to Delta. I think it's fine for it to use the same credentials as Delta itself.